setup-buildengine.sh

```diff
@@ -301,6 +299,7 @@ Options:
    -vm  | --vm-name <NAME>     create VM with <NAME> [$OPT_VM]
    --no-meta                   suppress creating meta data files with
                                'make-archive-meta.sh'
+    --basename <BASENAME>      basename for the result to targets and kits named as e.g. ${DEF_TARGET_BASENAME}-<version>-<arch>
```

Comment:
I would prefer: "base name for tooling and targets, must use alphanumeric characters only [$OPT_TARGET_BASENAME]" and wrapping the long line. The effect of the option will be available for review before it asks "Do you want to continue?", so I think it is not necessary to expand more on this here.